vue-project/src/models/image/ImageModel.ts

```diff
@@ -0,0 +1,52 @@
+// models/image/ImageModel.js
+import { LogError } from '@/helpers/LogHelper/LogError';
+import { ImageResolution } from './ImageResolution';
+export class ImageData {
+
+  targetWidth: number
+  targetHeight: number
+  currentImageSrc: string;
+  currentFileSize: number;
+  currentFileName: string;
+  currentResolution: ImageResolution;
+
+  constructor(file) {
+
+    this.currentResolution = new ImageResolution(1, 1);
```

Comment:
This is wrong.
why are you setting defaults at all?
You can set all of these when the image is loaded

Reply:
This is wrong.
why are you setting defaults at all?
You can set all of these when the image is loaded